.github/workflows/ci_master.yml

```diff
@@ -0,0 +1,58 @@
+name: nf-core CI for master
+
+# Run the workflow when:
+#  - dispatched manually
+#  - when a PR is opened or reopened to master branch
+#  - the head branch of the pull request is updated, i.e. if fixes for a release are pushed last minute to dev.
+on:
+  workflow_dispatch:
+    inputs:
+      testbranch:
+        description: "The specific branch you wish to utilize for the test execution."
+        required: true
+        default: "dev"
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+    branches:
+      - master
+  pull_request_target:
+    branches:
+      - master
+
+env:
+  NXF_ANSI_LOG: false
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  test-no-bed:
+    name: Run pipeline with test data with no BED
+    # Only run on push if this is the nf-core dev branch (merged PRs)
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        parameters:
+          - "--bed=`null`"
```

Comment:
Do you intend to test with a non-null bed file?

Reply:
Otherwise I would not put it in the matrix

Reply:
Since I need a full chromosome reference genome for paraphase for which DeepVariant takes ~45 min to run without a bed-file, I have a bed-file in the test profile by default (makes DeepVariant take a few minutes). 

I still want to have a test without a bed-file, so I compromised by adding it only when merging to master.